runtime/src/chplsys.c

```diff
@@ -201,6 +201,13 @@ static void getCpuInfo(int* p_numPhysCpus, int* p_numLogCpus) {
   if ((f = fopen("/proc/cpuinfo", "r")) == NULL)
     chpl_internal_error("Cannot open /proc/cpuinfo");
 
+  //
+  // If f is NULL, we should have exited by now, but Coverity doesn't
+  // seem to be catching this (anymore), so I'm adding an assertion
+  // to try and help it out.
+  //
+  assert(f != NULL);
```

Comment:
This should work fine.  I think in the past we've tried to quiet this one by adding an entry to the Coverity model file (util/devel/coverity/coverity_model.cpp) saying that chpl_internal_error() halts the program, but for some reason that didn't resolve the issue.  It might be worth trying to do that again, but for now this ought to work.